### PR TITLE
Users/anaggar/bug fix336374

### PR DESCRIPTION
**Pull request by AnubhavAggarwal**

### CODE REVIEW

File: Tasks/AzureFileCopy/AzureFileCopy.ps1 [hunk 1]
```diff
@@ -393,6 +393,11 @@ try
         Write-Verbose "Completed Get-EnvironmentResources cmdlet call for environment name: $environmentName with machine filter" -Verbose
     }
 
+    if ($resources.Count -eq 0)
```
Comment (pryenugu) [inline Tasks/AzureFileCopy/AzureFileCopy.ps1]: Should we handle null cases in $resources?

File: Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1 [hunk 2]
```diff
@@ -259,6 +259,11 @@ else
     Write-Verbose "Completed Get-EnvironmentResources cmdlet call for environment name: $environmentName with machine filter" -Verbose
 }
 
+if ($resources.Count -eq 0)
```
Comment (pryenugu) [inline Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1]: Should we handle null cases in $resources?

**Comment by pryenugu** [comment]
Looks good. Few comments.

**Comment by roniga** [comment]
Looks good.

File: Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1 (hunk 2 above)
Comment (AnubhavAggarwal) [inline Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1]: it cant be null

**Pull request merged**